analysis_tools/tracing/slog_trace_subscriber.h

```diff
@@ -0,0 +1,71 @@
+#include <cstring>
+#include <fstream>
+#include <map>
+#include <memory>
+
+#include "slog_cc/util/string_util.h"
+#include "slog_cc/context/subscribers.h"
+#include "slog_cc/context/context.h"
+
+namespace slog {
+
+struct SlogTraceSubscriberState {
```

Comment:
Would it help make things more readable to make this a private inner `struct` of the Subscriber type?